README.md

```diff
@@ -57,36 +51,47 @@ Package Name | Minimum Version
 
 Package Name | Minimum Version
 ------------ |:--------------:
-[sphinx](http://www.sphinx-doc.org/) | 1.4
-[inkscape](http://inkscape.org) |
```

Comment:
I'm going to make Inkscape a requirement for docs soon, so we should keep this.

Reply:
I just moved it to the 'core' requirements rather than the docs specific ones, as you need it to build gaffer itself, which is what tripped people up before.